tests/unit/lms/services/oauth1_test.py

```diff
@@ -46,46 +44,6 @@ def test_we_can_be_used_to_sign_a_request(self, service, application_instance):
         # one is present.
         assert "oauth_signature=" in auth_header
 
-    def test_sign(
```

Comment:
Removing the mock-only style of test,

see: https://github.com/hypothesis/lms/pull/6010#discussion_r1474280997